### PR TITLE
Prepare for release 0.1.0

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Switch all versions from 0.1.0-draft to 0.1.0.
Remove mentiones of draft / work-in-progress content. Update the README to mention 0.1.0 as latest release.

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has the [primer doc](https://github.com/cdevents/spec/blob/main/primer.md) been updated if a design decision is involved
- [x] Have the [JSON schemas](https://github.com/tektoncd/community/blob/main/standards.md#tests) been updated if the specification changed
- [x] Has spec version and event versions been updated according to the [versioning policy](https://github.com/cdevents/spec/blob/main/primer.md#versioning)
- [x] Meets the [CDEvents contributor standards](https://github.com/cdevents/.github/blob/main/docs/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,18 +38,19 @@ points for metrics.
 
 ## CDEvents Specification
 
-The [CDEvents Specification](./spec.md) is available on our
-[website](https://cdevents.dev/docs).
+The latest release of the specification is
+[v0.1](https://github.com/cdevents/spec/tree/0.1.0/spec.md).
```
Comment (e-backmark-ericsson) [inline README.md:42]: Minor, but I feel this section could be simplified to:
```suggestion
[v0.1.0](https://github.com/cdevents/spec/tree/0.1.0/spec.md), and you can continuously follow the latest updates of the specification on [the `main` branch](./spec.md)
```

File: primer.md [hunk 2]
```diff
@@ -409,6 +404,17 @@ There are two root use cases that we are considering:
   - Lead time for changes: the amount of time it takes a commit to get into production
   - Deployment frequency: how often an organization successfully releases to production
 
+## Terminology
+
+The [CDF Interoperability Special Interest Group][sig-interop] has produced a
+[table of the terms][tool-terms] used by different CI/CD tools and how they
+related to each other. The SIG is also working on distilling a recommended
```
Comment (e-backmark-ericsson) [inline primer.md:411]:
```suggestion
relate to each other. The SIG Interoperability is also working on distilling a recommended
```

File: primer.md [hunk 3]
```diff
@@ -409,6 +404,17 @@ There are two root use cases that we are considering:
...
+[shared terminology][shared-terms]. CDEvents strives to adopt to the shared
+terminology and collaborate with the SIG Interoperability.
+
+Work to align terms to those identified by the SIG will continue in upcoming
```
Comment (e-backmark-ericsson) [inline primer.md:415]:
```suggestion
Work to align terms to those identified by the SIG Interoperability will continue in upcoming
```

File: README.md [hunk 4]
```diff
@@ -38,18 +38,19 @@ points for metrics.
...
 
-The specification is currently draft and not ready for implementation.
-We are working towards our
-[first release v0.1](https://github.com/orgs/cdevents/projects/1).
+You can also preview the ongoing work on the [CDEvents Specification](./spec.md)
+on the `main` branch.
```
Comment (e-backmark-ericsson) [inline README.md:46]: I suggest to add a link to the primer here as well:
```suggestion
The concepts and ideas that have formed the current specification are outlined in the [CDEvents Primer](./primer.md)
```

File: primer.md [hunk 5]
```diff
@@ -207,7 +207,7 @@ format `vMajor.Minor.Patch`.
   identified by a change in the major version, for instance 0.1.2 -> 2.0.0
 
 While a version of the specification is work in progress, its version is tagged
-with an extra `-draft` at the end, for instance 0.1.0-draft.
+with an extra `-draft` at the end, for instance 0.1.0.
```
Comment (e-backmark-ericsson) [inline primer.md:210]: "draft" should not be excluded in this example :)
```suggestion
with an extra `-draft` at the end, for instance 0.1.0-draft.
```

File: primer.md [hunk 6]
```diff
@@ -187,7 +187,7 @@ The version of an event is included in its type. This allows for easy filtering
 of events of a specific version, by looking at a single field in the context.
 Examples of full event versions are:
 
-- `dev.cdevents.build.queued.0.1.0-draft`
+- `dev.cdevents.build.queued.0.1.0`
```
Comment (e-backmark-ericsson) [inline primer.md:190]: I think "draft" should be kept in this example 
```suggestion
- `dev.cdevents.build.queued.0.1.0-draft`
```

**Review by e-backmark-ericsson — CHANGES_REQUESTED** [review]

**Comment by afrittoli** [comment]
Thanks for the review, I addressed all comments / suggestions.

File: README.md [hunk 7]
```diff
@@ -39,11 +39,9 @@ points for metrics.
 ## CDEvents Specification
 
 The latest release of the specification is
-[v0.1](https://github.com/cdevents/spec/tree/0.1.0/spec.md).
-
-You can also preview the ongoing work on the [CDEvents Specification](./spec.md)
-on the `main` branch.
+[v0.1.0](https://github.com/cdevents/spec/tree/0.1.0/spec.md), and you can continuously follow the latest updates
```
Comment (e-backmark-ericsson) [inline README.md:42]: This sentence doesn't end correctly. Did you drop some of my previously proposed text?

**Review by e-backmark-ericsson — CHANGES_REQUESTED** [review]

**Review by e-backmark-ericsson — APPROVED** [review]

**Comment by e-backmark-ericsson** [comment]
Yippie! CDEvents v0.1.0 is now official!

**Pull request merged**